.github/workflows/publish_to_pypi_ci.yml

```diff
@@ -37,16 +37,16 @@ jobs:
                   name: python-package-distributions
                   path: ./wheelhouse/*.whl
 
-
-    publish-to-testpypi:
-      name: Publish Python distribution to TestPyPI
+          
+    publish-to-pypi:
+      name: Publish Python distribution to PyPI
       needs:
       - build
       runs-on: ubuntu-latest
```

Comment:
I forgot to mention in the previous PR that `ubuntu-22.04` is better than `ubuntu-latest` because the former ensures the runner is actually ubuntu 22.04.

Reply:
Thank you. I changed it to ubuntu-22.04 so that the runner does not change in a month.